src/app/helptext/services/components/service-afp.ts

```diff
@@ -49,11 +49,23 @@ afp_srv_map_acls_tooltip: T('Select mapping of permissions for\
  (default, Unix-style permissions), <b>None</b>,\
  or <b>Mode</b> (ACLs).'),
 afp_srv_map_acls_options : [
-  {label : 'Rights', value : 'RIGHTS'},
-  {label : 'None', value : 'NONE'},
-  {label : 'Mode', value : 'MODE'},
+  {label : T('Rights'), value : 'RIGHTS'},
+  {label : T('None'), value : 'NONE'},
+  {label : T('Mode'), value : 'MODE'},
 ],
 
+loglevel: {
+  placeholder: T('Log Level'),
+  tooltip: T('Choices are <i>Minimum, Normal, or Debug</i>.'),
```

Comment:
Suggestion to add more of a usage explanation to the help text:

```suggestion
  tooltip: T('Record AFP service messages up to the specified log level in the system log. By default, severe and warning level messages are logged.'),
```

Reply:
@Mrt134 , should we apply the same text to log-level in Services > SMB?

Reply:
I checked with Andrew and the text is generally applicable with one change as SMB has multiple log files:
```
Record SMB service messages up to the specified log level. By default, error and warning level messages are logged.

Samba messages are logged to various files:
<i>/var/log/samba4/log.smbd</i> : smbd process
<i>/var/log/samba4/log.winbindd</i> : winbindd parent process
<i>/var/log/samba4/log.wb-{DOMAIN} : forked winbindd child domain processes
<i>/var/log/samba4/log.nmbd</i> : NetBIOS name server
<i>/var/log/samba4/auth_audit.log</i> : authentication logs
```